Connected System Plug-in (CSP) Examples/Google Drive (OAuth,  Sending Files and Multiple Operations)/src/main/resources/appian-plugin.xml

```diff
@@ -1,4 +1,4 @@
-<appian-plugin name="My Corp Connected System - Google Drive Multi-Operation OAuth Sample" key="google-drive-multi-operation-sample">
+<appian-plugin name="My Corp Connected System - Google Drive Multi-Operation OAuth Sample" key="com.mycorp.google.drive">
```

Comment:
why google.drive instead of `google-drive`?

Reply:
Namespaces generally don't have dashes